Warnings with GCC 5.4.0

```
tests.cu(178): warning: result of call is not used
tests.cu(178): warning: result of call is not used
tests.cu: In function 'unsigned int error_checking(const char*, unsigned int)':
tests.cu:178:12: warning: ignoring return value of 'int system(const char*)', declared with attribute warn_unused_result [-Wunused-result]
   system(cmd);
```

--- a/tests.cu
+++ b/tests.cu
@@ -175,7 +175,10 @@ error_checking(const char* msg, unsigned int blockidx)
 		error_msg[ERR_MSG_LENGTH -1] = 0;
 		snprintf(cmd, CMD_LENGTH, "echo \"%s cuda_memtest errors found in %s[%d]\n%s\" |%s -s \" cuda_memtest errors found in %s[%d]\" %s",
 			 time_string(), hostname,gpu_idx, error_msg, MAILFILE, hostname,gpu_idx, emails );
-		system(cmd);
+		int sendMailResult = system(cmd);
+		if(sendMailResult != 0){
+			FPRINTF("ERROR: can not send the error report via mail to %s, '%s' returned an error\n", emails, MAILFILE);
+		}
 
 		firsttime = 0;
 		unreported_errors = 0;

